autogen/versions.tf.tmpl

```diff
@@ -19,9 +19,9 @@ terraform {
 
   required_providers {
 {% if beta_cluster %}
-    google-beta = "~> 2.18"
+    google-beta = "~> 3.3.0"
```

Comment:
Is anything we're doing breaking compatibility with 2.18?

If not we shouldn't *force* the upgrade to 3.3.0

```suggestion
    google-beta = ">= 2.18, <4.0.0"
```

Reply:
Done